sdk/OpenTap.Sdk.New/TranslateAction.cs

```diff
@@ -0,0 +1,226 @@
+//            Copyright Keysight Technologies 2012-2025
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Resources;
+using System.Resources.NetStandard;
+using System.Threading;
+using OpenTap.Cli;
+using OpenTap.Package;
+using OpenTap.Translation;
+
+namespace OpenTap.Sdk.New;
+
+/// <summary>
+/// This class contains helpful utilities for creating translations of packages.
+/// </summary>
+[Display("translate", Group: "sdk", Description: "Create a new translation template for a package.")]
+public class TranslateAction : ICliAction
```

Comment:
lets add an argument to specify language.

Reply:
This doesn't really make sense since we can only generate a template with neutral strings. I think creating files for other languages is in the domain of tooling such as [ResXManager](https://marketplace.visualstudio.com/items?itemName=TomEnglert.ResXManager) where the neutral language is used as the baseline for creating new translations.